### ISSUE TITLE
About Training dataset

### ISSUE THREAD

**Issue by sofiajensen** [ShineChen1024/oms-Diffusion#10]
Great work, much appreciated.

I'm curious about how you prepared the training dataset. I noticed that you trained it on Viton HD, so I'm guessing the following:

1.On the input side, you used cloth images, agnostic images, and masks from the Viton dataset. (1st, 2nd in attached image)
2. included fixed positive and negative prompts, such as "a photograph of a model" and "monochrome, low-res, bad anatomy, worst quality, low quality."
3.On the ground truth side, you used paired cloth images from the Viton dataset. (3rd one in attached image)

Could you please share more details on how you prepared the training dataset?

<img width="367" alt="image" src="https://github.com/ShineChen1024/oms-Diffusion/assets/145917077/11a34553-dcc8-459b-a7bc-298908a02e46">

**Comment by lucasmrl** [comment]
@sofiajensen did you get an answer?
